src/components/plans/chargeAccordion/ChargeWrapperSwitch.tsx

```diff
@@ -146,7 +148,7 @@ export const ChargeWrapperSwitch = memo(
             valuePointer={valuePointer}
           />
         )}
-        {localCharge?.chargeModel === ChargeModelEnum.Dynamic && <DynamicCharge />}
+        {localCharge?.chargeModel === 'dynamic' && <DynamicCharge />}
```

Comment:
I don't understand, why did we stop using the enum here?
I'd at least create a const for this. Let's avoid "magic strings / numbers"